src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -662,6 +663,17 @@ void regclass_graph_Model(py::module m) {
             :rtype: bool
         )");
 
+    function.def(
+        "clone_model",
```

Comment:
Why would we want to have this in the Model class? I don't think we should misalign it with C++ here, otherway it should be named as `Model.clone()`.  We can overload  `__copy__` for the Model, and allow user to call it like this:
`from copy import copy`
 `model_copy = copy(model)`

Reply:
Ok, that is a fair point (I missed it in the original comment, I am sorry!)

Reply:
Update: already fixed it, let's wait for CIs:)